pkg/tunnel/module.go

```diff
@@ -3,6 +3,8 @@ package tunnel
 import (
 	"context"
 	"fmt"
+	rcmgr "github.com/libp2p/go-libp2p/p2p/host/resource-manager"
```

Comment:
Golang package import order:
1. Built-in
2. Third parties
3. Project related

Otherwise CI will fail.

Reply:
Ok, I will change that